integration-test/src/test/java/com/datastrato/gravitino/integration/test/spark/SparkIT.java

```diff
@@ -260,6 +260,35 @@ void testListTable() {
     Assertions.assertThrows(NoSuchNamespaceException.class, () -> listTableNames("not_exists_db"));
   }
 
+  @Test
+  public void testAlterTableSetProperty() {
+    String tableName = "testSetProperty";
```

Comment:
please don't use Upper word for table name, rename to `test_property` because hive table name is not case sensitive. It will translate to `testsetproperty`

Reply:
fixed